libs/agora/ui/src/lib/components/footer/footer.component.ts

```diff
@@ -62,9 +62,9 @@ export class FooterComponent implements OnInit {
   }
 
   getSiteVersion() {
-    return this.sha
-      ? `${this.configService.config.appVersion}-${this.sha}`
-      : this.configService.config.appVersion;
+    let appVersion = formatGitTag(this.configService.config.appVersion);
+    appVersion = appVersion.replace(/-rc\d+$/, '');
```

Comment:
Can we remove the second line? It looks like `formatGitTag` handles replacing the rc suffix

```suggestion
    const appVersion = formatGitTag(this.configService.config.appVersion);
```

Reply:
Ah yes, I had already refactored this.  Thanks!